From 3996192f44b11902f4d8ea9af297dd0773cedb91 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Tue, 6 Aug 2019 22:42:44 +0200
Subject: [PATCH] clamav: update to version 0.100.3

Fixes CVEs:
0.100.1
- CVE-2017-16932
- CVE-2018-0360
- CVE-2018-0361
0.100.2
- CVE-2018-15378
- CVE-2018-14680
- CVE-2018-14681
- CVE-2018-14682
0.100.3
- CVE-2019-1787
- CVE-2019-1788
- CVE-2019-1789

Use HTTPS in URL

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 net/clamav/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/net/clamav/Makefile b/net/clamav/Makefile
index 53b5787ef..87d14c06a 100644
--- a/net/clamav/Makefile
+++ b/net/clamav/Makefile
@@ -8,7 +8,7 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=clamav
-PKG_VERSION:=0.100.0
+PKG_VERSION:=0.100.3
 PKG_RELEASE:=1
 
 PKG_LICENSE:=GPL-2.0
@@ -17,7 +17,7 @@ PKG_MAINTAINER:=Marko Ratkaj <marko.ratkaj@sartura.hr> \
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://www.clamav.net/downloads/production/
-PKG_HASH:=c5c5edaf75a3c53ac0f271148fd6447310bce53f448ec7e6205124a25918f65c
+PKG_HASH:=5160cdf876c761a7d611f158b7c774ebbbe25856b7c230d9c4e142015e8d3024
 
 PKG_BUILD_PARALLEL:=1
 PKG_INSTALL:=1
@@ -31,7 +31,7 @@ define Package/clamav/Default
   CATEGORY:=Network
   SUBMENU:=Web Servers/Proxies
   TITLE:=ClamAV
-  URL:=http://www.clamav.net/
+  URL:=https://www.clamav.net/
 endef
 
 define Package/clamav
-- 
2.20.1

